test/DebugInfo/dead-store-elimination.sil

```diff
@@ -0,0 +1,26 @@
+// RUN: %target-sil-opt %s -dead-store-elimination -enable-sil-verify-all | %FileCheck %s
+
+// REQUIRES: swift_in_compiler
+
+sil_stage canonical
+
+import Swift
+import Builtin
+
+// We should be able to replace the local store with a debug_value.
+// The dead store elimination pass is written in Swift, and should call
+// salvage debug info when removing the store.
+
+// CHECK-LABEL: @trivial_local_dead_store
+// CHECK: debug_value %{{[0-9]+}} : $Int, var, name "a"
+// CHECK: return
+sil hidden @trivial_local_dead_store : $@convention(thin) () -> () {
+bb0:
+  %0 = alloc_stack $Int, var, name "a"
+  %1 = integer_literal $Builtin.Int64, 1
+  %2 = struct $Int (%1 : $Builtin.Int64)
```

Comment:
This test will probably not work on 32-bit platforms. Could you rewrite it to hardcode an Int32?